src/api.md

```diff
@@ -8,3 +8,4 @@ Any missing parameter, or parameter with an invalid format, should result in a J
 
 - "hexadecimal-encoded" designates a binary value encoded as hexadecimal. The value must either be empty, or start with `"0x"` and contain an even number of characters.
 - "SCALE-encoded" designates a value encoded using [the SCALE codec](https://docs.substrate.io/v3/advanced/scale-codec/).
+- "Merkle value" as decribed in the [Polkadot spec](https://spec.polkadot.network/chap-state#defn-merkle-value).
```

Comment:
I don't think that's really the right place, as the Merkle value is used by a single function in the entire API. I'd just put that sentence in `chainHead_unstable_follow.md`.

Reply:
You also mention it in `chainHead_unstable_storage`. IMO would be nice to just use the "global" glossary.